### DISCUSSION TITLE
Transaction Stuck. Waiting for a relayer to process your transfer.

### DISCUSSION THREAD

**Discussion by shomroni** [wormhole-foundation/wormhole#1612] [Problems / bugs] [answered]
Trying to bridge UST Classic (wormhole) from Ethereum to Terra Classic. Stuck on Waiting for a relayer to process your transfer for a while now.
Any ideas what is wrong?

**Comment by evan-gray** [comment, marked as answer]
While it is possible that no relayer picked up your transaction, you can always manually redeem it via [Portal](https://www.portalbridge.com/#/redeem)
